package.json

```diff
@@ -81,7 +81,7 @@
     "webpack-merge": "^4.2.1"
   },
   "engines": {
-    "node": ">=6"
+    "node": ">=10"
```

Comment:
I think we should actually just go to node 12 here. Node 10 expires in ~30 days or so.

Reply:
Updated to 12.